pkg/cli/middleware/middleware.go

```diff
@@ -0,0 +1,27 @@
+package middleware
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+func WithBeforeFns(fns ...cli.BeforeFunc) cli.BeforeFunc {
```

Comment:
Can you add godoc to these two functions describing the motivation?

Reply:
Done